breathecode/jobs/tests/tasks/tests_async_run_spider.py

```diff
@@ -0,0 +1,56 @@
+from django.utils import timezone
+from datetime import timedelta
+from unittest.mock import MagicMock, call, patch
+from ..mixins import JobsTestCase
+from breathecode.tests.mocks import (
+    REQUESTS_PATH,
+    apply_requests_post_mock,
+)
+
+from ...tasks import async_run_spider
+
+JOBS = [{
+    'Searched_job': 'junior web developer',
+    'Job_title': 'Pentester Cybersecurity',
+    'Location': 'Remote (Chile, Venezuela)',
+    'Company_name': 'Repite Employer',
+    'Post_date': 'November 05, 2021',
+    'Extract_date': '2022-01-30',
+    'Job_description': 'Vuln exploitation Security reports',
+    'Salary': 'Not supplied',
+    'Tags': ['back-end', 'cybersecurity', 'english', 'pentesting', 'python'],
+    'Apply_to': 'https://www.url.com/1',
+    '_type': 'dict'
+}, {
+    'Searched_job': 'junior web developer',
+    'Job_title': 'Pentester Cybersecurity',
+    'Location': 'Remote (Chile)',
+    'Company_name': 'Repite Employer',
+    'Post_date': 'November 05, 2021',
+    'Extract_date': '2022-01-30',
+    'Job_description': 'Vuln exploitation Security reports',
+    'Salary': 'Not supplied',
+    'Tags': ['back-end', 'cybersecurity', 'english', 'pentesting', 'python'],
+    'Apply_to': 'https://www.url.com/2',
+    '_type': 'dict'
+}]
+
+spider = {'name': 'getonboard', 'zyte_spider_number': 3, 'zyte_job_number': 0}
+zyte_project = {'zyte_api_key': 1234567, 'zyte_api_deploy': 223344}
+platform = {'name': 'getonboard'}
+
+
+class RunSpiderTaskTestCase(JobsTestCase):
+    @patch('breathecode.jobs.actions.run_spider', MagicMock())
```

Comment:
The same review that `breathecode/jobs/tests/tasks/tests_async_fetch_sync_all_data.py`

Reply:
when you refere to same revew, what is you want to say? I don't undertand that.